pkg/skaffold/deploy/kustomize.go

```diff
@@ -121,33 +117,48 @@ func (k *KustomizeDeployer) Deploy(ctx context.Context, out io.Writer, builds []
 		event.DeployInfoEvent(errors.Wrap(err, "could not fetch deployed resource namespace."+
 			"This might cause port-forward and deploy health-check to fail."))
 	}
+
+	if err := k.kubectl.Apply(ctx, textio.NewPrefixWriter(out, " - "), manifests); err != nil {
+		event.DeployFailed(err)
+		return NewDeployErrorResult(errors.Wrap(err, "kubectl error"))
+	}
+
+	event.DeployComplete()
+	return NewDeploySuccessResult(namespaces)
+}
+
+func (k *KustomizeDeployer) renderManifests(ctx context.Context, out io.Writer, builds []build.Artifact, labellers []Labeller) (deploy.ManifestList, error) {
```

Comment:
I'm not sure why `Render` doesn't have `[]Labeller` as parameter. I think that's something that should get added in the future so the behaviour is similar to `Deploy`.

Reply:
it should have it. looks like maybe I forgot to put it in there when I added the boilerplate a while back.